kairon/shared/data/processor.py

```
@@ -9191,16 +9191,12 @@ def fetch_action_logs_for_parallel_action(self, trigger_id: str, bot: str) -> Li
-            raise AppException(f"Parallel action not found")
-
-        # Get all logs matching the action names and bot
         logs = (
             ActionServerLogs
-            .objects(trigger_info__trigger_id=trigger_id, bot=bot, action__in=parallel_action.actions)
+            .objects(trigger_info__trigger_id=trigger_id, bot=bot)
             .order_by("-timestamp")
             .as_pymongo()
         )
+        if not logs:
+            raise AppException(f"Loga for Actions in Parallel Action not found")
```

Ruff (0.11.9)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)